### ISSUE TITLE
There are 1600 pieces of my dataset. Why only 200 pieces are displayed here? v6.0

### ISSUE THREAD

**Issue by pengchen0429** [ultralytics/yolov5#9811]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

![1](https://user-images.githubusercontent.com/86482968/195974669-453e3459-9925-4e17-badc-3fbf52da857b.jpg)

**Comment by glenn-jocher** [comment]
Your validation set has 200 images.

**Comment by pengchen0429** [comment]
Thank you, the previous 1600 training images were used for training, did you use 200 validation sets for model validation after each round of training?

**Comment by pengchen0429** [comment]
So why not show the number of my training set and choose to show the number of validation sets? Looking forward to your answer

**Comment by pengchen0429** [comment]
> 您的验证集有 200 张图像。
那么为什么不显示我的训练集的数量而选择显示验证集的数量呢？期待你的回答

**Comment by glenn-jocher** [comment]
@pengchen0429 your splits are clearly shown at the beginning of training.

Val step obviously shows val images and metrics.
